main.js

```diff
@@ -163,7 +163,8 @@ let electricSign = {
   x: 0,
   y: 0,
   w: 140,
-  h: -140
+  h: -140,
+  obst: false
```

Comment:
Let's name this property something easier to read, e.g. `is_background` to indicate whether the obstacle is really part of the background and not a collidable obstacle.